site/en/docs/privacy-sandbox/protected-audience-api/index.md

```diff
@@ -486,6 +486,14 @@ and membership is removed when users clear their site data.
 
 {% endDetails %}
 
+{% Details %}
+{% DetailsSummary %}
+### Are the Protected Audience worklets cached by the browser?
+{% endDetailsSummary %}
+
+The resources that contain the Protected Audience worklets - the buyer bid generation worklet, seller ad scoring worklet, buyer reporting worklet, and seller scoring worklet - are cached by the browser. You can use the `Cache-Control` header to control the caching behavior.
```

Comment:
```suggestion
The resources that contain the Protected Audience worklets—the buyer bid generation worklet, seller ad scoring worklet, buyer reporting worklet, and seller scoring worklet—are cached by the browser. You can use the `Cache-Control` header to control the caching behavior.
```

Reply:
The cascading adjectives make it a bit confusing, are they buyer bid-generation worklet / seller ad-scoring worklet or buyer-bid generation worklet / seller-ad scoring worklet?

Reply:
It's "buyer bid-generation worklet" but I think the most correct way to phrase this is "buyer's bid generation worklet".  

I'll update this to: 
—the buyer's bid generation and reporting worklets, and seller's ad scoring and reporting worklets—